tests/test_parser.py

```diff
@@ -1361,17 +1356,6 @@ def test_bad_2():
         )
 
 
-def test_error_on_relation():
```

Comment:
As everything can be written on a single line, this is now allowed: "test" is a reference to a variable.
I think it would be safer to enforce 1 statement per line:
```
"""
Host.provider [1] -- Provider
test
"""
```

Reply:
While (as I said before) I do agree in a sense, it's not that simple. We still want to support
```
MyEntity(
    x=1
)
```
I don't think it's feasible to implement something like this now.